package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "reddit-flair-apps",
   "private": true,
-  "version": "1.5.8",
+  "version": "1.5.9",
```

Comment:
We should do proper semantic versioning

major.minor.patch

Major: breaking api change
Minor: new feature
Patch: bug fix